lib/engine-addon.js

```diff
@@ -66,7 +65,7 @@ function buildConfigTree(prefix) {
   // Include a module that reads the engine's configuration from its
   // meta tag and exports its contents.
   var configContents = this.getEngineConfigContents();
-  var configTree = writeFile(prefix + '/config/environment.js', configContents);
```

Comment:
Can you explain this one to me? Are we relying on the fact that it gets returned from the addon to namespace it?